MapboxDirections/MBRouteOptions.swift

```diff
@@ -225,10 +225,9 @@ open class RouteOptions: NSObject, NSSecureCoding {
 
         includesExitRoundaboutManeuver = decoder.decodeBool(forKey: "includesExitRoundaboutManeuver")
         
-        guard let locale = decoder.decodeObject(of: NSLocale.self, forKey: "locale") as Locale? else {
-            return nil
+        if let locale = decoder.decodeObject(of: NSLocale.self, forKey: "locale") as Locale? {
```

Comment:
Since the locale property is optional, you can set it directly without the if statement.